examples/example32/generate_weights.sh

```diff
@@ -1,4 +1,25 @@
 #!/bin/bash
+# This script is used to extract the projectability from
+# the output of projwfc.x (proj.out) and it works only
+# for this specific system with 21 bands.
+
+# Remove temporary files if exist
+if [ -f "e.dat" ]; then
+	rm e.dat;
+fi
+
+if [ -f "p.dat" ]; then
+        rm p.dat;
+fi
+
+if [ -f "p_vs_e.dat" ]; then
```

Comment:
I am going to approve this, but I mention that this is not really correct - this file is anyway regenerated, while the file appended with `>>` is instead `tmp.dat`, this is the one that should be removed...

Also, you can just `rm -f FILENAME` to remove it and not complain if it does not exist, no need to check.

Finally, below if there is a problem, better to use `exit 1` instead of `exit`, to mark an unsuccessful exit.